samples/ORKCatalog/ORKCatalog/Configuration/ORKCatalog-Shared.xcconfig

```diff
@@ -11,7 +11,7 @@ ORK_CATALOG_BUILD_NUMBER_CI_ = $(ORK_CATALOG_BUILD_NUMBER_CI_FALSE)
 
 ORK_CATALOG_BUNDLE_ID_BASE = com.example.researchkit-samplecode.$(PRODUCT_NAME:rfc1034identifier)
 
-ORK_CATALOG_BUNDLE_ID = $(ORK_CATALOG_BUNDLE_ID_CI_$(CI)) // ORK_CATALOG_BUNDLE_ID_CI_TRUE or ORK_CATALOG_BUNDLE_ID_CI_
-ORK_CATALOG_BUNDLE_ID_CI_TRUE = $(CI_BUNDLE_ID)$(ORK_CATALOG_CI_BUNDLE_ID_SUFFIX) // if CI_BUNDLE_ID is defined (presumably by CI) just use it, allow appending an optional suffix
+ORK_CATALOG_BUNDLE_ID = $(ORK_CATALOG_BUNDLE_ID_CI_$(USE_CI)) // ORK_CATALOG_BUNDLE_ID_CI_TRUE or ORK_CATALOG_BUNDLE_ID_CI_
+ORK_CATALOG_BUNDLE_ID_CI_TRUE = $(USE_CI_BUNDLE_ID)$(ORK_CATALOG_CI_BUNDLE_ID_SUFFIX) // if CI_BUNDLE_ID is defined (presumably by CI) just use it, allow appending an optional suffix
```

Comment:
This might just be here in the example, but I think that we might need to check the diff to ensure that we do not introduce some regressions here. We set `CI` (https://github.com/StanfordBDHG/.github/blob/main/.github/workflows/archive.yml#L70) to true and not `USE_CI`. Might also affect some other build related scripts and elements in here, this is just the position where I first noticed it. This one might not even be relevant as this is the example but it would be great to check this in the different projects.

Reply:
I'm not sure why Apple made this change within the file you mentioned, they are even inconsistent within the same file: https://github.com/StanfordBDHG/ResearchKit/blob/94d6a12b9c157a0f4716577a8a96ce5ddb16ccd7/samples/ORKCatalog/ORKCatalog/Configuration/ORKCatalog-Shared.xcconfig#L7

For now, I replaced the `USE_CI` with just `CI` (as we pass this flag via the `xcodebuild archive` cmd but I'm not sure what else I should do here.

Reply:
Sounds good, that should work! As far as I understand ORKCatalog is an own thing by itself put there for demonstration purposes and the overall setup is a bit unfortunate as the CI and the detailed setup is not open source and the project is not really maintained with a community aspect in mind. I think it is good to merge as it is.